### ISSUE TITLE
Decontamination unit weirdness with a hopper

### ISSUE THREAD

**Issue by idkw** [MJRLegends/ExtraPlanets#467]
### ---Issue Report---
**Have you checked Closed Issues (if applicable & PLEASE CHECK BEFORE POSTING):**
Yes

### Description of Issue

The decontamination unit can be powered by placing batteries in the batery slot either : 

- by placing the battery with a player interaction putting the item in the slot
![image](https://user-images.githubusercontent.com/7012331/63136137-e950cc00-bfd0-11e9-97ce-6c81a5d8691f.png)

- by inserting the item in the battery slot with a pipe from another mod or a vanilla hopper
![image](https://user-images.githubusercontent.com/7012331/63136149-f7065180-bfd0-11e9-807c-ce3b1bf057b5.png)

The tile entity accepts any item that matches `ItemElectricBase.isElectricItem` : 
https://github.com/MJRLegends/ExtraPlanets/blob/3fe42f1b2ecf355acdf6b41e759266b05ba11f3d/src/main/java/com/mjr/extraplanets/tileEntities/machines/TileEntityBasicDecontaminationUnit.java#L191

The check is implemented in Galacticraft Core with : 

https://github.com/micdoodle8/Galacticraft/blob/a7cfed1364db24bb03192005434f67c9c68144c5/src/main/java/micdoodle8/mods/galacticraft/core/energy/item/ItemElectricBase.java#L213

```
    public static boolean isElectricItem(Item item)
    {
        if (item instanceof IItemElectricBase)
        {
            return true;
        }

        if (EnergyConfigHandler.isIndustrialCraft2Loaded())
        {
            if (item instanceof ic2.api.item.ISpecialElectricItem)
            {
                return true;
            }
        }

        return false;
    }
```

So any IC2 electric item is valid according to this check when IC2 is loaded.

Indeed I can add a Lapotron Crystal from IC2 :

![image](https://user-images.githubusercontent.com/7012331/63136671-667d4080-bfd3-11e9-8c2a-fde2c5d06546.png)

However when working with the container, with pipes or hoppers, it only accepts galacticraft or Extra Planets batteries. IC2 electric items such as Lapotron Crystal are not inserted by the hopper :

![image](https://user-images.githubusercontent.com/7012331/63136705-90366780-bfd3-11e9-9930-d59b5ac5d729.png)

### Steps to Reproduce Issue

When Industrial craft is Loaded you can place a Lapotron Crystal in the energy slot manually but it doesn't work with a hopper pointing into the central base block of the decontamination unit.

**Version of Mod using (PLEASE DONT USE 'Latest' OR 'Newest'):**

ExtraPlanets-1.12.2-0.5.5.jar
GalacticraftCore-1.12.2-4.0.2.230.jar

**Where did this bug happen:** [Single Player or Server]

Server And Single Player

**Comment by MJRLegends** [comment]
Sorry for the delay, the hopper for this machine/unit should be disabled anyways. Sorry :D

**Comment by MJRLegends** [comment]
Updates released for MC 1.12.2, 1.11.2, 1.10.2. 

MC 1.8.9 will come later as got more work to do for that version :D

**Comment by MJRLegends** [comment]
Update released for MC 1.8.9
